website/blog/2022-07-25-coreapp.md

```diff
@@ -0,0 +1,128 @@
+---
+title: "Streamlining app creation with React Native for Windows CoreApp"
+author: Alexander Sklar
+authorURL: https://twitter.com/alexsklar
+description: "Streamlining app creation with React Native for Windows CoreApp"
+is_blog: true
+published_date: July 25, 2022
+---
+
+React Native builds on top of the native platform for every OS it runs on; on Windows, this translates to producing a Universal Windows Platform (UWP) app. 
+
+We heard from some of our customers that build times for these types of apps can be pretty long - especially so when building the React Native for Windows framework from source. To mitigate this problem we offer pre-built binary distributions of the framework in the form of NuGet packages. This works pretty well but has a few shortcomings:
+
+1) Even when not building the framework, building a UWP requires several build steps behind the scenes (the steps below are what happens for C++/WinRT apps):
+    - the MIDL compiler is run to turn IDL files into Windows Metadata - this is required to produce information used by the XAML compiler
+    - the XAML compiler is invoked to convert XAML markup files into C++ code
+    - the MDMerge tool runs to merge all Windows Metadata files from the application
+    - C++/WinRT is invoked to turn Windows Metadata into C++ projection headers as well as component sources
+    - Only then, the C++ compiler and linker run on the app
+2) Native modules don't currently fully support consuming the RNW NuGet package that the app they are getting built into is using; there is work in progress to unlock this scenario. 
```

Comment:
I'm not sure what this sentence is saying. Is it:
" Native modules can't consume the prebuilt RNW NuGet package when they are included in an app that depends on it (though there is work outstanding to fix this)"?

Would recommend linking to the tracking issue.

Reply:
actually this might not be true anymore after re-reading this comment:
https://github.com/microsoft/react-native-windows/issues/7227#issuecomment-1136441556

@jonthysell is that right? modules created in 0.68 don't need to do anything to use their app's RNW nuget (instead of building RNW from source - and they don't need to build RNW from source anymore?)

Reply:
removed that whole mention, assuming the comment about NuGet not working for modules is no longer accurate